app/controllers/users/passwords_controller.rb

```diff
@@ -12,6 +12,7 @@ class PasswordsController < Devise::PasswordsController
     end
 
     # rubocop:disable AbcSize,MethodLength
+    # TODO(sbc): Refactor to address rubocop warnings
```

Comment:
This should be cleaned up (and I did in a PR that we're still deciding whether or not to merge), so the offense should not be disabled.